README - INSTALL INSTRUCTIONS/QBOX/README.md

```diff
@@ -1,5 +1,11 @@
-# qbx_core Resource Edits
-Search for `createCharacter` in client/character.lua and replace with
+
+# QBX Core
+
+## `client/character.lua`
```

Comment:
Having this as the first line in the code field (outcommented) is a better practice.